app/workers/certificates.php

```diff
@@ -160,7 +129,6 @@ public function perform()
             ],
             'domain' => $domain->get(),
             'issueDate' => \time(),
```

Comment:
It will be a good option to have the expiration date in the DB, might be good for displaying in the UI. It might also be a good idea to store the schedule check. This will potentially allow us to find domains that have missed that check for any reason (redis failure / power shutdown - using `checkDate>1 day`)

Reply:
To clarify, you'd like to store the `60day+1` renewal check, as well as the daily check that is performed as a part of this PR?

Reply:
expirationDate = current time + 90 days after a successful creation 
checkDate = current time + 1 day

Rethinking about it, if we have `issueDate` we can calculate `expirationDate` without needing to store it.

Reply:
> Rethinking about it, if we have issueDate we can calculate expirationDate without needing to store it.
Works for me - certbot doesn't have a ready-to-parse command for this.

> checkDate = current time + 1 day
Ah, I understand better - thanks for clarifying.